src/Pages/HomePage.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const HomePage: React.FC = () => {
+  return (
+    <h1 className="title">Home page</h1>
+  );
```

Comment:
Use the container div here
```suggestion
  return (
    <div className="container">
      <h1 className="title">Home page</h1>
    </div>
  );
```